frontend/app/src/i18n/en.json

```diff
@@ -334,7 +334,7 @@
         "buychat_q": "How do I buy CHAT tokens?",
         "buychat_a": "Right now you must first <a target=\"_blank\" href=\"https://www.finder.com/uk/how-to-buy-internet-computer\">buy ICP</a> which you can then swap for CHAT on various Decentralised Exchanges (DEXes).<ul><li><a target=\"_blank\" href=\"https://avjzx-pyaaa-aaaaj-aadmq-cai.raw.ic0.app/ICDex/CHAT/ICP\">ICDex</a></li><li><a target=\"_blank\" href=\"https://info.icpswap.com/swap/pool/details/ne2vj-6yaaa-aaaag-qb3ia-cai\">ICPSwap</a></li><li><a target=\"_blank\" href=\"https://app.infinityswap.one/tokens/2ouva-viaaa-aaaaq-aaamq-cai\">InfinitySwap</a></li></ul>DFINITY are adding support for the <a target=\"_blank\" href=\"https://www.rosetta-api.org/\">Rosetta API</a> for ICRC1 tokens (which includes CHAT) making them much easier to integrate with Centralised Exchanges (CEXes). Once this has happened it is likely CHAT will be available to trade on CEXes for ICP or USDT.",
         "diamond_q": "What is 💎 Diamond Membership?",
-        "diamond_a": "You can upgrade to Diamond membership to have access to extra or enhanced features within OpenChat. <a href=\"?diamond=\">See here</a> for a feature comparison and to upgrade.",
+        "diamond_a": "You can upgrade to Diamond membership to have access to extra or enhanced features within OpenChat. <a href=\"/?diamond=\">See here</a> for a feature comparison and to upgrade.",
```

Comment:
This change navigates the user away from wherever they are

Reply:
Yes - it's a bit annoying. That link is not going to work from a landing page. It will only happen if you're clicking from the FAQ, but I guess it is annoying. Will have to think about how to make that work.

Reply:
Probably the only real solution is to extract the diamond features to work as a landing page too and then link to the landing page version from the faq. Can still keep the modal version for upgrade and for the `Tell me about Diamond` shortcut. Bit of a faff.

Reply:
Done